config/zones/TH.yaml

```diff
@@ -19,18 +19,108 @@ capacity:
+        datetime: '2006-01-01'
+        source: EPPO 2006
+        value: 571
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2007-01-01'
+        source: EPPO 2007
+        value: 571
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2008-01-01'
+        source: EPPO 2008
+        value: 570
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2009-01-01'
+        source: EPPO 2009
+        value: 560
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2010-01-01'
+        source: EPPO 2010
+        value: 551
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2011-01-01'
+        source: EPPO 2011
+        value: 530
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2012-01-01'
+        source: EPPO 2012
+        value: 530
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2013-01-01'
+        source: EPPO 2013
+        value: 532
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2014-01-01'
+        source: EPPO 2014
+        value: 532
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2015-01-01'
+        source: EPPO 2015
+        value: 507
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2016-01-01'
+        source: EPPO 2016
+        value: 493
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2017-01-01'
+        source: EPPO 2017
+        value: 471
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2018-01-01'
+        source: EPPO 2018
+        value: 459
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2019-01-01'
+        source: EPPO 2019
+        value: 445
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2020-01-01'
+        source: EPPO 2020
+        value: 442
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2021-01-01'
+        source: EPPO 2021
+        value: 433
+      - _comment: Gross Energy Generation of EGAT and Net Energy Generation of IPP, SPP, and VSPP
+        datetime: '2022-01-01'
+        source: EPPO 2022
+        value: 409
```

Comment:
I would just add the source as EPPO or EPPO 2023 since it was released this year and then add a top level sources object like this for example:
```yaml
sources:
  EU-ETS, ENTSO-E 2021:
    link: https://drive.google.com/file/d/15UeUHyhcjVw8fGLc0Zx_HELSx20cw_-e/view?usp=share_link
```